cluster/manifests/prometheus/ingress.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
   hosts:
     - system-prometheus.{{ .Values.hosted_zone }}
+    - system-prometheus-{{.Cluster.Alias}}.{{ .Values.hosted_zone }}
```

Comment:
For the most cases `.Cluster.Alias` is the prefix of the `.Values.hosted_zone`.
Maybe we can make it 
```
if .Values.hosted_zone startswith .Cluster.Alias then
  system-prometheus.{{ .Values.hosted_zone }}
else
  system-prometheus-{{.Cluster.Alias}}.{{ .Values.hosted_zone }}
end
```
WDYT?

Reply:
OTOH this will require conditional logic on client side to build up hostname so maybe its not worth it.